web/src/app/services/user.service.ts

```diff
@@ -197,4 +199,7 @@ export class UserService {
       .subscribe(() => this.router.navigate(['/dashboard'])
       );
   }
+  getUsersWithMaxFollowers() {
+    return this.http.get(PROVIDERS.TRIBE_COUNT_API);
```

Comment:
How is the `http` request protected?

Reply:
Planning to add auth headers. Will do it in the next PR.

Reply:
Ok 👍 , might be easier to have it as a callable function?